logicrunner/builtin/contract/member/member.go

```diff
@@ -407,106 +416,109 @@ func (m *Member) createMember(name string, key string, burnAddress string) (*mem
 }
 
 // Migration methods.
-func (m *Member) migration(txHash string, burnAddress string, amount big.Int, unHoldDate time.Time) (string, error) {
+func (m *Member) depositMigration(txHash string, burnAddress string, amount *big.Int) error {
 	rd := rootdomain.GetObject(m.RootDomain)
 
 	// Get migration daemon members
-	migrationDaemonMembers, err := rd.GetMigrationDaemonMembers()
+	migrationDaemonMembers, err := rd.GetActiveMigrationDaemonMembers()
```

Comment:
Should we use map[ref]bool instead?

Reply:
No, because we need index. index = migration daemon machine (real machine)